internal/shim-sgx/src/handler/enarx.rs

```diff
@@ -1,8 +1,14 @@
 // SPDX-License-Identifier: Apache-2.0
 
+use sallyport::request;
+use sallyport::syscall::SGX_DUMMY_QUOTE;
+use sallyport::syscall::SGX_DUMMY_TI;
+use sallyport::syscall::SYS_ENARX_GETATT;
 use sallyport::syscall::{BaseSyscallHandler, EnarxSyscallHandler, SGX_QUOTE_SIZE, SGX_TECH};
 use sallyport::untrusted::{UntrustedRef, UntrustedRefMut, ValidateSlice};
-use sallyport::Reply;
+
+use crate::uarch::REPORT;
+use crate::uarch::TARGETINFO;
```

Comment:
It's very easy to confuse these for constants, how about we rename these using "normal" Rust names on import to prevent this confusion? https://github.com/enarx/enarx/pull/1360#discussion_r802702282

Reply:
Originally they have been named correctly:
https://github.com/enarx/sgx/blob/e4d5cbe232b51b40c8003dedc7226e167e5224de/src/attestation_types/report.rs#L169-L172
and
https://github.com/enarx/sgx/blob/e4d5cbe232b51b40c8003dedc7226e167e5224de/src/attestation_types/ti.rs#L10-L13

nicely aligned and checked with `testaso!`

Reply:
These names are aligned with SDM (see the comment in the file header).

Reply:
Inline comments in uarch structure fields make notorious to modify anything as it makes the struct more cluttered. Sometimes comments are harmful.

Reply:
@jarkkojs I do not suggest renaming the types, but rather names they are imported as

Reply:
I updated struct names.